app/uk/gov/hmrc/hec/models/CTStatusResponse.scala

```diff
@@ -25,20 +25,9 @@ final case class CTStatusResponse(
   ctutr: CTUTR,
   startDate: LocalDate,
   endDate: LocalDate,
-  status: CTStatus,
-  accountingPeriods: List[AccountingPeriod]
+  latestAccountingPeriod: Option[AccountingPeriod]
```

Comment:
just to double check, was it not an array of accounting period?

Reply:
the API will return us an array, but we're only ever gonna be interested in the latest one in the search period. I thought doing it like this where the BE does this work means that the FE can just reuse this case class to store in session without storing a bunch of other accounting periods which will never get used